test/pagespecs/AccessibilityStatementPageSpec.scala

```diff
@@ -19,20 +19,4 @@ package pagespecs
 import testsupport.ItSpec
 import testsupport.stubs._
 
-class AccessibilityStatementPageSpec extends ItSpec {
-
-  def beginJourney(): Unit = {
-    AuthStub.authorise()
-    TaxpayerStub.getTaxpayer()
-    IaStub.successfulIaCheck
-    GgStub.signInPage(port)
-    startPage.open()
-    startPage.clickOnStartNowButton()
-  }
-
-  "check link and content are correct" in {
-    beginJourney()
-    accessibilityStatementPage.clickOnAccessibilityStatementLink()
-    accessibilityStatementPage.assertPageIsDisplayed
-  }
-}
+class AccessibilityStatementPageSpec extends ItSpec {}
```

Comment:
Suggest here to either:
a) Remove this suite entirely and remove the `accessibilityStatementPage` as well
or b) update the existing test to just check that the accessibility link is present and points at the right place